packages/utils/src/common/to_async_iterator.ts

```diff
@@ -23,8 +28,18 @@ export async function toAsyncIterator<T extends IDecodedMessage>(
     options
   );
 
+  const isWithTimeout = Number.isInteger(iteratorOptions?.timeoutMs);
```

Comment:
if timeout value is not specified, what happens in the browser?

Reply:
No timeout will be applied and execution will wait till resolved.